static/app/views/integrationPipeline/githubInstallationSelect.tsx

```diff
@@ -18,12 +22,20 @@ type Installation = {
 };
 
 type GithubInstallationProps = {
+  has_scm_multi_org: boolean;
   installation_info: Installation[];
+  organization_slug: string;
 };
 
-export function GithubInstallationSelect({installation_info}: GithubInstallationProps) {
-  const [installationID, setInstallationID] = useState<SelectKey>(-1);
+function GithubInstallationSelect({
+  installation_info,
+  has_scm_multi_org,
+  organization_slug,
+}: GithubInstallationProps) {
+  const [installationID, setInstallationID] = useState<SelectKey>('-1');
```

Comment:
The `-1` feels a bit arbitrary (unless it actually does work on the backend, in which case we need a comment explaining that). I think to help with future debugging, being more explicit would be nice:

```ts
const [installationID, setInstallationID] = useState<SelectKey | null>(null);
```

Reply:
Yeah the -1 is arbitrary. It was chosen because how the fe & be communicate is via query parameters. So the be pipeline will read the existence of `chosen_installation_id` to know the user just finished the organization selection UI step. It's currently default set to -1 so that the Select component will default choose the 'Install on another organization' option.

Tbh I'd prefer a comment + leaving the default as -1 🤷‍♀️  lmk thoughts